packages/jest-mock/src/__tests__/index.test.ts

```diff
@@ -1215,6 +1215,30 @@ describe('moduleMocker', () => {
     expect(fn.getMockName()).toBe('jest.fn()');
   });
 
+  test('after mock reset, the object should return to its original value', () => {
+    const myObject = {bar: () => 'bar'};
+
+    const barStub = jest.spyOn(myObject, 'bar');
```

Comment:
this should use the `moduleMocker` created at the top of this test file, and not the `jest` object

Reply:
Oops, fixed! 😃